pkg/auth/dependency/provider/anonymous/anonymous.go

```diff
@@ -0,0 +1,48 @@
+package anonymous
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/skygeario/skygear-server/pkg/core/db"
+)
+
+const providerPassword string = "anonymous"
```

Comment:
`providerPassword` => `providerAnonymous`

Reply:
Since it is private, I can be just `providerName`. But for consistent with provider/password. it can be providerAnonymous.